src/plugins/undoRedo/undoRedo.js

```diff
@@ -267,6 +271,8 @@ UndoRedo.Action.prototype.redo = function() {};
  * Change action.
  *
  * @private
+ * @param changes
```

Comment:
I'm curious why in this file almost all @params are without type and descriptions?

Reply:
Resolved within https://github.com/handsontable/handsontable/pull/6647/commits/55ab30a38a6b414bb1204617b1989f89169664c8.